Hoxtygen/next-fullstack
# feat: get agent listing enquiries (HOM-76)

## What does this PR do?
* Get all listing enquiries of agents

## Description of the tasks to be completed
* Implement UI to display enquiries
* Integrate backend endpoint to fetch data

## How should this be manually tested?
* Clone the app
* Open a terminal on your PC and run git clone <repo_url>
* `cd` into the project directory and run `yarn install`.
* After package installation run `yarn dev`
* Open a tab on your browser and navigate to http://localhost:3000/auth/signup to sign up or http://localhost:3000/auth/login to log in if you already have an account
* Click on the `Messages` link in the sidebar to see `messages`

## What are the relevant Jira board stories?
[HOM-76](https://wasiu-dev.atlassian.net/jira/software/projects/HOM/boards/2?selectedIssue=HOM-76)

[HOM-76]: https://wasiu-dev.atlassian.net/browse/HOM-76?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
